src/nucleic_acid/nucleic_acid.jl

```diff
@@ -49,6 +66,24 @@ end
 end
 
 
+function P_generic(mod::NASM, t::Float64, scaled::Bool)
```

Comment:
If you leave this as P_generic(mod::NASM, t, scaled::Bool), Julia should figure out what to do. It would then make the implementation for arrays of time unnecessary in lines 78-84